blog/2025-04-08-agama-13.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Releasing version 13
+description: Summary of the Agama 13 release
+discussion_id: 69
+tags:
+  - release
+---
+
+Time flies and more than a month has passed since the announcement of Agama 12. Fortunately the YaST
+Team has not been idle and we have a new version of Agama to present. Say hello to Agama 13,
+including some additions to the web interface and many improvements for unattended installations.
+
+{/* truncate */}
+
+## Setting the hostname {#hostname}
+
+Let's start with a feature that some people would consider rather basic or, at least, something easy
+to implement - the ability to set a name for the machine (ie. hostname) during installation. The
+naked truth is that it took us some time to open that can of worms because it is a rather
+complicated topic.
+
+First of all, there is nothing like an unique hostname since a machine can have different
```

Comment:
"a unique" because pronounced  you-nique which is not a vowel... yes, English and its weirdnesses ;-)